cpp-ch/local-engine/Storages/SubstraitSource/SubstraitFileSource.cpp

```diff
@@ -240,6 +240,21 @@ DB::ColumnPtr FileReaderWrapper::createConstColumn(DB::DataTypePtr data_type, co
     return column;
 }
 
+DB::ColumnPtr FileReaderWrapper::createColumn(DB::DataTypePtr data_type, size_t rows, const String & value)
+{
+    if (StringUtils::isNullPartitionValue(value))
+    {
+        auto nested_type = DB::removeNullable(data_type);
+        auto column = nested_type->createColumnConstWithDefaultValue(rows);
+        return DB::ColumnNullable::create(column, DB::ColumnUInt8::create(rows, 1));
```

Comment:
use makeNullable

Reply:
makeNullable seems not to be what i need.  It cannnot gen a totally null column which null map is all filled with 1